packages/web3.js-plugin-fee-estimation/src/plugin.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { beforeAll, describe, expect, test } from 'vitest'
+import Web3, { Contract, FMT_BYTES, FMT_NUMBER } from 'web3'
+import {
+  l2StandardBridgeABI,
+  l2StandardBridgeAddress,
+  optimistABI,
+  optimistAddress,
+} from '@eth-optimism/contracts-ts'
+
+import OptimismFeeEstimationPlugin from './plugin'
+
+const provider = process.env.VITE_L2_RPC_URL ?? 'https://mainnet.optimism.io'
+
+describe('OptimismFeeEstimationPlugin', () => {
+  let web3: Web3
+
+  beforeAll(() => {
+    web3 = new Web3(provider)
+    web3.registerPlugin(new OptimismFeeEstimationPlugin())
+  })
+
+  test('should be registered under .op namespace', () =>
+    expect(web3.op).toBeDefined())
```

Comment:
```suggestion
    expect(web3.op).toMatchInlineSnapshot())
```
Nit: maybe do this so we can at a glance see every method registered on web3.op for anyone using this test as documentation